pkg/cmd/pulumi/import_test.go

```diff
@@ -456,10 +459,23 @@ func TestImportFileMarshal(t *testing.T) {
   ]
 }
 `
-	var buffer bytes.Buffer
-	enc := json.NewEncoder(&buffer)
-	enc.SetIndent("", "  ")
-	err := enc.Encode(importFile)
-	require.NoError(t, err)
-	assert.Equal(t, expected, buffer.String())
+		var buffer bytes.Buffer
+		enc := json.NewEncoder(&buffer)
+		enc.SetIndent("", "  ")
+		err := enc.Encode(importFile)
+		require.NoError(t, err)
+		assert.Equal(t, expected, buffer.String())
```

Comment:
Minor and can clean this up later but given this encode/equals check is the same for both test cases this would be slightly nicer as a table driven test.

Reply:
Top one uses equals whereas the bottom is using contains. Could be reduced to table tests, but the amount of indirection added to support just these two seems not worth it at the moment.

Reply:
Fair enough.